kaska/utils.py

```diff
@@ -175,10 +177,15 @@ def save_output_parameters(time_grid, observations, output_folder, parameter_nam
     for (param, data) in zip(parameter_names, output_data):
         for band, tstep in enumerate(time_grid):
             this_date = tstep.strftime("%Y%j")
-            if chunk is None:
-                outfile = output_folder/f"s2_{param:s}_A{this_date:s}.tif"
-            else:
-                outfile = output_folder/f"s2_{param:s}_A{this_date:s}_{chunk:s}.tif"
+            
+            # Compose the output file name from pattern (satellite),
+            # parameters, dates and, optionally, chunk
+            extra = ""
+            if chunk is not None:
+                extra = f"_{chunk:s}"
+            
+            outfile = output_folder/f"{fname_pattern:s}_{param:s}_A{this_date:s}{extra}.tif"
```

Comment:
I believe we can remove all the `:s` from there. In the case of the date, I would say `{tstep:%Y%j}` to don't create the intermediate variable.

Reply:
The intermediate variable is used elsewhere. The instances of `:s` have been removed.